script.js

```diff
@@ -39,30 +33,34 @@ const validateAge = (age) => {
 
     return isValidAge
 }
-const validateheight = (height) => {
+const validateHeight = (height) => {
     const isValidHeight  = height > 0 && height < 3
     
     return isValidHeight
 }
-const validateSkinColor = (skinColor) => {
-    const isValidSkinColor = skinColor !== undefined
-    return isValidSkinColor
-}
 
-const skinColorConvert = (skinColorData) => {
-    const skinColorObject = {
+const getColorByValue = (skinColorData) => {
+    const skinColorMap = {
         "BR": "Branco",
         "NG": "Negro",
         "PD": "Pardo"
     };
-    const skinColor = skinColorObject[skinColorData];
+    const skinColor = skinColorMap[skinColorData];
     return skinColor;
 }
-const showToast = (message,styleToast) =>{
+const showToast = ({message,type}) =>{
 
     const toast = document.createElement('span')
     toast.innerText = message
-    toast.className = styleToast
+    
+    switch(type){
```

Comment:
O que acontece se o tipo do `toast` for diferente de `success` ou de `error`? O toast deveria aparecer na tela?